### ISSUE TITLE
api v2 doesn't retrieve scores with scorev2 mod

### ISSUE THREAD

**Issue by AkinariHex** [ppy/osu-web#8309]
I'm creating a team sheet on my website that fetch automatically the scores from players recent scores, but seems that scoreV2 plays aren't submitted since 2020. I can't retrieve them either with api v1 and v2.

Will be possible to re-enable the scoreV2 submission or to retrieve these scores from api?

**Comment by peppy** [comment]
I believe this was disabled for an actual reason, will need to look into why and if we can work around it.

**Comment by nanaya** [comment]
I vaguely remember it's about tournament training thing

**Comment by AkinariHex** [comment]
> I vaguely remember it's about tournament training thing

I don't want to say something wrong, but I can barely remember that years ago tournament players used to watch scorev2 scores to see which maps their opponents were good to play and because of this scorev2 was disabled when OWC started (I don't remember the year)

**Comment by Negaosu** [comment]
yes it's because people used bots to track scores submitted in private mp practice lobbies since they would submit
